package.json

```diff
@@ -17,16 +17,21 @@
   },
   "scripts": {
     "build": "node ./scripts/build.js",
-    "prepublishOnly": "node ./scripts/prepublish.js",
-    "test-watch": "karma start testing/karma.conf.js --auto-watch",
-    "test": "karma start testing/karma.conf.js --single-run",
-    "test-debug": "karma start testing/karma.conf.js --browsers Chrome",
+    "build-storybook": "build-storybook",
+    "prepublishOnly": "pinst --disable && node ./scripts/prepublish.js",
```

Comment:
The [Husky documentation](https://typicode.github.io/husky/#/?id=install) recommended adding these `pinst` commands in the pre- and post-publish hooks to disable `husky` when publishing the package. Without this, downstream users of the package will get an error after installing the package, which would be dumb. (The `postinstall` hook would be run on this package when we run `yarn install` in `vets-website`, for example. This would attempt to install the git hooks in the `node_modules/@department-of-veterans-affairs/component-library` and would result in an error because `husky` is a `devDependency` because...y'know...it's a dependency for development.)

The alternative is to remove the `postinstall` NPM hook completely, but then we risk contributors not installing the git hooks and getting the benefit there.